examples/monitor/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+
+services:
+
+  mosec:
+    image: python:3.9-slim-buster
+    container_name: mosec
+    ports:
+      - "8000:8000"
+      - "5000:5000"
+    volumes:
+      - ../python_side_metrics.py:/root/python_side_metrics.py
+    entrypoint: ["/bin/bash", "-c","pip install -i https://pypi.tuna.tsinghua.edu.cn/simple -U mosec prometheus_client && python /root/python_side_metrics.py"]
```

Comment:
Don't do this. You can build from a local Dockerfile.

BTW, remove the mirror after testing.